.circleci/images/cuberite-circleci/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:bionic
+
+# Install build dependencies
+RUN apt-get update && \
+	apt-get install -y git lua5.1 cmake clang-6.0 clang-tidy-6.0 python-pip && \
+	pip install PyYAML 
```

Comment:
Already in the Ubuntu repository as [python-yaml](https://packages.ubuntu.com/bionic/python-yaml).